types/openai.ts

```diff
@@ -42,4 +43,10 @@ export const OpenAIModels: Record<OpenAIModelID, OpenAIModel> = {
     maxLength: 96000,
     tokenLimit: 32000,
   },
+  [OpenAIModelID.GPT_35_TURBO_16K]: {
+    id: OpenAIModelID.GPT_35_TURBO_16K,
+    name: "GPT 3.5 Turbo",
+    maxLength: 12000,
+    tokenLimit: 4000,
```

Comment:
```suggestion
    maxLength: 48000,
    tokenLimit: 16000,
```
I think tokenLimit should be 16000 since the 16k in the model name signifies 16k tokens. maxLength seems to be 3x that in the other examples so just guessing 48000 for that here

Reply:
good catch!